packages/boxel-ui/addon/src/components/filter-list/usage.gts

```diff
@@ -0,0 +1,83 @@
+import { fn } from '@ember/helper';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import FreestyleUsage from 'ember-freestyle/components/freestyle/usage';
+
+import FilterList, { type Filter } from './index.gts';
+
+export default class FilterListUsage extends Component {
+  filters: Filter[] = [
+    {
+      displayName: 'All Apps',
+      query: {
+        filter: {
+          eq: {
+            _cardType: 'Apps',
+          },
+        },
+      },
+    },
+    {
+      displayName: 'All Cards',
+      query: {
+        filter: {
+          eq: {
+            _cardType: 'Card',
+          },
+        },
+      },
+    },
+    {
+      displayName: 'Person',
+      query: {
+        filter: {
+          eq: {
+            _cardType: 'Person',
+          },
+        },
+      },
+    },
+    {
+      displayName: 'Pet',
+      query: {
+        filter: {
+          eq: {
+            _cardType: 'Pet',
+          },
+        },
+      },
+    },
+  ];
+  @tracked activeFilter: Filter = this.filters[0]!;
+
+  <template>
+    <FreestyleUsage @name='Filter List'>
+      <:example>
+        <FilterList
+          @filters={{this.filters}}
+          @activeFilter={{this.activeFilter}}
+          @onClickFilter={{fn (mut this.activeFilter)}}
+        />
+      </:example>
```

Comment:
Maybe the usage could show an actual example where filtering does something?

Reply:
Updated.


https://github.com/user-attachments/assets/1a220d79-9c5a-46c1-b03d-d5a863b910f1